packages/lodestar-cli/src/lodecli/cmds/beacon/cmds/run/run.ts

```diff
@@ -24,7 +25,12 @@ export async function run(options: Arguments<IBeaconArgs & Partial<IBeaconNodeOp
   options = deepmerge(defaultOptions, options) as Arguments<IBeaconArgs & Partial<IBeaconNodeOptions>>;
 
   const peerId = await readPeerId(options.network.peerIdFile);
+  // read local enr from disk
   options.network.discv5.enr = await readEnr(options.network.enrFile);
+  // read bootstrap enrs from configuration
+  options.network.discv5.bootEnrs = options.network.discv5.bootEnrs
+    ? options.network.discv5.bootEnrs.map(e => ENR.decodeTxt(e as unknown as string))
```

Comment:
what is type of e though?

Reply:
actual type is string, perhaps we can wait for https://github.com/ChainSafe/discv5/pull/54 and a release, then we won't need this